From: Simon McVittie <smcv@collabora.com>
Date: Fri, 16 Apr 2021 10:41:47 +0100
Subject: revokefs: Use the right variable name in an error message

scan-build detected that response_size is uninitialized here, presumably
a typo for response_data_size.

Signed-off-by: Simon McVittie <smcv@collabora.com>
Forwarded: https://github.com/flatpak/flatpak/pull/4227
---
 revokefs/writer.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/revokefs/writer.c b/revokefs/writer.c
index d514016..3e821d9 100644
--- a/revokefs/writer.c
+++ b/revokefs/writer.c
@@ -876,7 +876,7 @@ do_writer (int basefd_arg,
 
       if (response_data_size < 0 || response_data_size > MAX_DATA_SIZE)
         {
-          g_printerr ("Invalid response size %ld", response_size);
+          g_printerr ("Invalid response size %ld", response_data_size);
           exit (1);
         }
 
